api/src/main/java/org/openmrs/module/locationbasedaccess/common/UserControlAdvice.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.locationbasedaccess.common;
+
+import org.aopalliance.intercept.MethodInterceptor;
+import org.aopalliance.intercept.MethodInvocation;
+import org.openmrs.Patient;
+import org.openmrs.Person;
+import org.openmrs.User;
+import org.openmrs.PersonAttribute;
+import org.springframework.aop.framework.ReflectiveMethodInvocation;
+import java.util.Set;
+
```

Comment:
What is the use of this class?

Reply:
It's a common class for the AOP Methods. We can use for users and patients while accessing the service methods.